spec/features/logging/view_auth_requests_for_a_username_spec.rb

```diff
@@ -51,6 +55,23 @@
       click_on "Show logs"
     end
 
+    context 'when an organsiation with no ips searches logs' do
+      before do
+        sign_in_user admin_user_2
+        visit new_logs_search_path
+        choose "Username"
+        click_on "Go to search"
+        fill_in "Username", with: search_string
+        click_on "Show logs"
+      end
+
+      let(:search_string) { "AAAAAA" }
+
+      it "displays no results results " do
+        expect(page).to have_content("\"#{username}\" isn't reaching the GovWifi service")
```

Comment:
strictly speaking, you should use `search_string` instead of `username`

Reply:
Nice catch